ui/src/__tests__/spec/wdio.conf.js

```diff
@@ -1,73 +1,145 @@
+'use strict';
+const fs = require('fs');
 const path = require('path');
-const BROWSERS = require('./libs/browsers');
-const browsers = (process.env.BROWSERS || 'chrome').split(',');
-
-const debug = require('debug')('AthenzUI:functional:wdioconf');
+const mkdirp = require('mkdirp');
+const exec = require('child_process').exec;
+const appConfig = require('../../config/config')();
 const {
-    BUILD_NUMBER,
-    MAX_INSTANCES: maxInstances = 20,
-    PARENT_TUNNEL = 'yahoo_general',
-    RETRIES,
-    SAUCE_ACCESS_KEY,
-    SAUCE_USERNAME,
-    SCREWDRIVER,
-    TEST_DIR = path.resolve(process.cwd(), 'artifacts'),
-    TUNNEL_IDENTIFIER = 'tunnel-yahoo-corp',
-    OKTA_IT,
-    OKTA_AT,
-    COOKIE_DOMAIN,
-    INSTANCE,
+  BUILD_NUMBER,
+  SD_JOB_NAME,
+  WORK_DIR,
+  TUNNEL_IDENTIFIER = appConfig.tunnelIdentifier,
+  PARENT_TUNNEL = appConfig.parentTunnel,
 } = process.env;
+let athenzDomain;
+let athenzService;
+let sdv4FunctionalJob = SD_JOB_NAME && SD_JOB_NAME === appConfig.devSdJobName;
+
+if (process.env.INSTANCE) { // executing functional test pointing at dev environment
+  athenzDomain = 'athenz.k8s';
```

Comment:
we have this information as part of default-config.js, lets use the value from there

Reply:
hmm I can't seem to find the values in default-config.js, could you give me a line reference?

Initially I saw that there's `athenzDomainService: 'athenz.k8s.athenz-ui-development'` in `extended-config.js` however, i want to split up the athenz and service to be able to execute functional test locally as `athenz.k8s` pointing to dev environment and `athenz.dev` pointing at local environment

Reply:
I was referring to the athenzDomainService and then extract value from it or add 2 more config options, just looking to reduce the number of lines we need to change with a service change, not a blocker